nuntium/templates/nuntium/profiles/messages_per_instance.html

```diff
@@ -26,7 +26,7 @@
     {% blocktrans count message_count=writeitinstance.message_set.count %}
     <strong>1</strong> message in your WriteIt
```

Comment:
The screenshot makes me think that I left this out because another change already did that … but I can't find that right now